packages/features/ee/billing/credit-service.test.ts

```diff
@@ -41,6 +52,39 @@ vi.mock("../workflows/lib/reminders/reminderScheduler", () => ({
   cancelScheduledMessagesAndScheduleEmails: vi.fn(),
 }));
 
+// Create a testable version of CreditService that exposes protected methods
```

Comment:
I had to create this testable version of CreditService, because we need to mock protected mehtods like `_getTeamWithAvailableCredits`, ... and this was the only way I've found to expose them.